docs/flow/monitoring/debugging.md

```diff
@@ -6,3 +6,56 @@ weight: 300
 ---
 
 # Debugging
+
+There are two primary ways to debug issues with Grafana Agent Flow:
+
+1. Through the Grafana Agent Flow UI
+2. Through examining logs
+
+## Grafana Agent Flow UI
```

Comment:
It makes sense for us to put screenshots somewhere in this section, but I'd like to hold off on doing that until the UI is merged in main just in case details change during review.